meta/estao-immo/estao/de.yml

```diff
@@ -1,20 +1,66 @@
 de:
-    title: estao® Immobilienlösung für Hausverwalter und Wohnungsunternehmen
+    title: estao®️ Immobilienlösung für Vermieter und Makler
```

Comment:
da scheint sich ein unsichtbares Zeichen eingeschlichen zu haben?

Reply:
Uj ... wurde das ® nicht übernommen?

Reply:
<img width="1036" alt="Bildschirmfoto 2019-06-05 um 22 03 36" src="https://user-images.githubusercontent.com/1073273/58986495-ca1e0d00-87dd-11e9-91df-fc2b67228b3a.png">

Nein, bei mir in Safari zeigt es ein unsichtbares Zeichen an

Reply:
Das unsichtbare Zeichen ist nun auch rein.